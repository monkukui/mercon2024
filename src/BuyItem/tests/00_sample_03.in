15
4 20 71
10 10 57
2 20 50
6 19 31
5 6 25
3 2 79
5 16 9
2 5 20
1 3 90
9 13 91
9 9 67
4 7 87
10 14 75
5 15 64
6 3 42
