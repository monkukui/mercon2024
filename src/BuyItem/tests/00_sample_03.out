13
2 3
3 1
5 2
6 5
7 4
9 9
10 10
13 9
14 10
15 5
16 5
19 6
20 4
